kiosk/sources/extensions/personal_kiosk.move

```diff
@@ -51,12 +51,19 @@ module kiosk::personal_kiosk {
     public fun new(
         kiosk: &mut Kiosk, cap: KioskOwnerCap, ctx: &mut TxContext
     ): PersonalKioskCap {
-        assert!(kiosk::has_access(kiosk, &cap), EWrongKiosk);
+        new_(kiosk, cap, sender(ctx), ctx)
+    }
 
-        let owner = sender(ctx);
+    /// Wrap the KioskOwnerCap making the Kiosk "owned" and non-transferable.
+    /// The `PersonalKioskCap` is returned to allow chaining within a PTB, but
+    /// the value must be consumed by the `transfer_to_sender` call in any case.
+    public fun new_(
```

Comment:
I don't think this function should be public.

Reply:
Thanks for catching this, you are definitely right.